vcsutils/webhookparser/github.go

```diff
@@ -60,11 +60,14 @@ func (webhook *GitHubWebhook) parsePushEvent(event *github.PushEvent) *WebhookIn
 
 func (webhook *GitHubWebhook) parsePrEvents(event *github.PullRequestEvent) *WebhookInfo {
 	var webhookEvent vcsutils.WebhookEvent
-	if event.GetAction() == "opened" {
-		webhookEvent = vcsutils.PrCreated
-	} else if event.GetAction() == "synchronize" {
+	switch event.GetAction() {
+	case "opened", "reopened":
+		webhookEvent = vcsutils.PrOpened
+	case "synchronize", "edited":
```

Comment:
Are you sure we want to listen to the "edited" event? If I'm not mistaken, this event is triggered when a pull request description is changed.

Reply:
[Documentation](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request) says that changing the base branch triggers `synchronize` event. Although when I tested changing the base branch, I got `edited` event instead, so we need to listen to this event as well to capture changes of the base branch for PR.

Reply:
OK, thanks!